exercises/matching-brackets/runtests.jl

```diff
@@ -0,0 +1,23 @@
+using Test
```

Comment:
Please add a comment `# canonical data version: x.y.z` to the very top of the file. Replace `x.y.z` with the canonical data version you used.